sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ComputedPropertiesCodeSnippet.java

```diff
@@ -36,7 +36,7 @@ public void createContainerWithComputedProperties() {
                 )
         );
         containerProperties.setComputedProperties(computedProperties);
-        database.createContainer(containerProperties);
+        database.createContainer(containerProperties).subscribe();
```

Comment:
I am wondering should we use block() here?

Reply:
+ 1 below as well

Reply:
We can, but since this is just code snippet, I don't want customers to learn bad coding practices for async